python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,99 +3,278 @@
 import os
 import sys
 import platform
+import subprocess
+
 platform_system = platform.system().lower()
-exit_code = 0
-
-try:
-    import zlib
-    zlib
-except:
-    print '"zlib" missing.'
-    exit_code = 1
-
-try:
-    import _hashlib
-    _hashlib
-except:
-    print 'standard "ssl" missing.'
-    exit_code = 1
-
-try:
-    from OpenSSL import SSL, crypto, rand
-    SSL
-    crypto
-    rand
-except:
-    print '"OpenSSL" missing.'
-    exit_code = 1
-
-try:
-    import Crypto
-    Crypto
-except:
-    print '"PyCrypto" missing.'
-    exit_code = 1
-
-
-try:
-    import crypt
-    crypt
-except:
-    print '"crypt" missing.'
-    exit_code = 1
-
-try:
-    import pysqlite2
-    pysqlite2
-except:
-    print '"pysqlite2" missing.'
-    exit_code = 1
-
-try:
-    import setproctitle
-    setproctitle
-except:
-    print '"setproctitle" missing.'
-    exit_code = 1
-
-try:
-    from ctypes import CDLL
-    CDLL
-except:
-    print '"ctypes - CDLL" missing.'
-    exit_code = 1
-
-try:
-    from ctypes.util import find_library
-    find_library
-except:
-    print '"ctypes.utils - find_library" missing.'
-    exit_code = 1
-
-try:
-    from Crypto.PublicKey import _fastmath
-    _fastmath
-except:
-    print 'Crypto.PublicKey._fastmath missing. No GMP?'
-    exit_code = 1
-
-# Windows specific modules.
-if os.name == 'nt':
+
+
+def set_expected_deps():
```

Comment:
method is called set_ but in fact it gets.

it does not return a list for every supported os. but the list of approved deps for **current** os